demos/capi/README.md

```diff
@@ -0,0 +1,89 @@
+# C and C++ API inference demo (C/C++)
```

Comment:
Should we name it C++ api? It is only C API, but apps can be made in C++ lang. @dtrawins @rasapala

Reply:
I think it should be named C API. C++ is compatible with C API's but naming this C++ may be misleading as it could suggest there is C++ version of API that would be easier to use by C++ developers.

To sum up I would name it C API but mention in release notes etc, that it can be used in C/C++ applications

Reply:
Ok.